### PR TITLE
Add Holybro M8N M9N GPS

### PR DESCRIPTION
**Pull request by vincentpoont2**

### CODE REVIEW

**Comment by vincentpoont2** [comment]
@hamishwillee Can you look into this one? This is different from https://github.com/PX4/PX4-user_guide/pull/2229
Thanks!

File: en/gps_compass/gps_holybro_m8n_m9n.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  
+  ![Holybro M8N with Pixhawk GPS1 connector](../../assets/hardware/gps/holybro_gps_pinout.jpg)
+- **SKU12014 Holybro M8N 2nd GPS (JST GHR1.25m m 6pin cable)**
+
+  This JST GH 6Pin connector can be used on Pixhawk Series 6pin `UART`, `I2C` or `GPS2` input port as secondary GPS.
+  
+  ![Holybro M8N with Pixhawk 2nd GPS connector](../../assets/hardware/gps/holybro_gps_pinout3.jpg)
+- **SKU12013 Holybro M8N GPS for Pix32 (Molex 1.25mm 6pin & 4pin & 3pin cable)**
+
+  These Molex 1.25mm 6pin & 4pin & 3pin connectors are for Pix32 `Switch`, `GPS`, and `I2C` input ports (not for Pix32 v5 or v6).
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_m8n_m9n.md:52]: What is this platform? i.e. it says " (not for Pix32 v5 or v6).". Does the platform it connects to support PX4?

File: en/gps_compass/gps_holybro_m8n_m9n.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Holybro M8N & M9N GPS
+
+The GPS includes either a M8N or M9N UBLOX module, IST8310 compass, tri-colored LED indicator, and a safety switch.
+
+There are 3 different connector options, allowing easy connection to flight controllers that are compliant with the Pixhawk-connector standard as either a primary or secondary GPS.
+This module ships with a baud rate of 38400 5Hz.
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_m8n_m9n.md:6]: What does "38400 5Hz." mean?

**Comment by hamishwillee** [comment]
Hi @vincentpoont2 
I've reviewed, but please be aware I'm holiday on and off for the rest of this month. Around this week though if you want quick turnaround.

File: en/gps_compass/gps_holybro_m8n_m9n.md (hunk 1 above)
Comment (vincentpoont2) [reply en/gps_compass/gps_holybro_m8n_m9n.md:52]: This is the Pix32.
https://docs.px4.io/main/en/flight_controller/holybro_pix32.html

pix32 v5 and v6 has different connector type.

File: en/gps_compass/gps_holybro_m8n_m9n.md (hunk 2 above)
Comment (vincentpoont2) [reply en/gps_compass/gps_holybro_m8n_m9n.md:6]: 38400 Hz

File: en/gps_compass/gps_holybro_m8n_m9n.md (hunk 1 above)
Comment (hamishwillee) [reply en/gps_compass/gps_holybro_m8n_m9n.md:52]: Thanks. Makes sense.
```suggestion
  These Molex 1.25mm 6pin & 4pin & 3pin connectors are for the [Holybro pix32 Flight Controller](../flight_controller/holybro_pix32.md) `Switch`, `GPS`, and `I2C` input ports (not for the Pix32 v5 or v6).
```

File: en/gps_compass/gps_holybro_m8n_m9n.md (hunk 2 above)
Comment (hamishwillee) [reply en/gps_compass/gps_holybro_m8n_m9n.md:6]: Cool! 
```suggestion
This module ships with a baud rate of 38400 Hz.
```

**Comment by hamishwillee** [comment]
Thanks. Merged!

**Pull request merged**